core/src/main/java/com/sequenceiq/distrox/v1/distrox/StackOperations.java

```diff
@@ -93,6 +95,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(StackOperations.class);
 
+    private static final int UPGRADE_NODE_COUNT_LIMIT = 200;
```

Comment:
This could be a configuration similar to these in application.yml:
```
  limit:
    nodeCountLimits:
      cm: 400
      infrastructure: 500
```